frappe/desk/calendar.py

```diff
@@ -13,6 +14,8 @@ def update_event(args, field_map):
 	w = frappe.get_doc(args.doctype, args.name)
 	w.set(field_map.start, args[field_map.start])
 	w.set(field_map.end, args.get(field_map.end))
+	date_obj = datetime.strptime(args.get(field_map.schedule_date), "%Y-%m-%d %H:%M:%S")
```

Comment:
This ideally shouldn't be required IMO. If it's DateTime field the appropriate conversion is handled automatically.

Reply:
removed the conversion